bigchaindb/upsert_validator/validator_utils.py

```diff
@@ -12,9 +12,8 @@ def encode_validator(v):
     # NOTE: tendermint expects public to be encoded in go-amino format
     pub_key = PubKey(type='ed25519',
                      data=bytes.fromhex(ed25519_public_key))
-    return Validator(pub_key=pub_key,
-                     address=b'',
-                     power=v['power'])
+    return ValidatorUpdate(pub_key=pub_key,
```

Comment:
What exactly is this change from Validator to ValidatorUpdate doing?

Reply:
API changed. to add a Validator, ValidatorUpdate has to be called from now on.